packages/ui-components/src/utils/responsive-utils/use-breakpoint.ts

```diff
@@ -0,0 +1,23 @@
+import { useDimensions } from '@webb-dapp/react-environment/layout';
+import { useMemo } from 'react';
+
+import { size } from './break-points.util';
+
+export default () => {
```

Comment:
Instead of the default export consider adding a function with a name
`export function useBreakpoint() {

...
}`